InvenTree/InvenTree/views.py

```diff
@@ -904,3 +904,18 @@ def get_context_data(self, **kwargs):
         """
 
         return ctx
+
+
+class NotificationsView(TemplateView):
+    """ View for showing notifications
+    """
+
+    template_name = "InvenTree/notifications/notifications.html"
+
+    def get_context_data(self, **kwargs):
+
+        ctx = super().get_context_data(**kwargs).copy()
+
+        # ctx['settings'] = InvenTreeSetting.objects.all().order_by('key')
```

Comment:
does get_context_data need to be reimplemented here, now that this line is commented out?